stdlib/ast.pyi

```diff
@@ -1186,12 +1186,18 @@ class Slice(_Slice):
         ) -> Self: ...
 
 class ExtSlice(slice):  # deprecated and moved to ast.py if sys.version_info >= (3, 9)
```

Comment:
I went ahead and added it for all the classes deprecated in 3.9. I didn't bother making a 3.8 branch without `@deprecated` since that's just one version and EOL anyway. The descriptions are all taken from the docstrings in `ast.py`.